app/assets/javascripts/sitewide/atwho_res.js.erb

```diff
@@ -578,3 +570,11 @@ var SmartAnnotation = (function() {
   return publicApi;
 
 })();
+
+
+// initialize the smart annotations
+(function initSmartAnnotation() {
+  $(document).on('focus', '[data-atwho-edit]', function() {
+    SmartAnnotation.init(this);
```

Comment:
Just double-checking: does this re-initialize `atwho` each time you focus the input field, or is the initialization done only once?

I just don't want this whole JS code to be initialized multiple times, because it can potentially query server via AJAX multiple times for each inputted character, which is pretty bad.